pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.contrastsecurity</groupId>
     <artifactId>contrast-sdk-java</artifactId>
-    <version>2.23-SNAPSHOT</version>
+    <version>3.0</version>
```

Comment:
Keep it 3.0-SNAPSHOT. The release process will release 3.0 and move the development version to 3.1-SNAPSHOT.

Reply:
Shoot, got to excited moving to v3